tools/readme_linter/assert.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"sort"
+
+	"github.com/yuin/goldmark/ast"
+)
+
+//type for all linter assert methods
+type T struct {
+	filename       string
+	markdown       []byte
+	newlineOffsets []int
+
+	fails int
+}
+
+// called by all assert functions that involve a node
+func (t *T) printFailedAssertf(n ast.Node, format string, args ...interface{}) {
+	t.printFile(n)
+	fmt.Printf(format+"\n", args...)
+	//t.printRule(3)
```

Comment:
This is commented out, but it was there to print out the markdown rule itself? If you are going to leave this commented out can you add a comment explaining why?

Reply:
I added a command line flag for this in 529ad78